src/cms/views/dashboard/dashboard_view.py

```diff
@@ -36,4 +42,23 @@ def get(self, request, *args, **kwargs):
         """
 
         val = "To be defined"
-        return render(request, self.template_name, {"key": val, **self.base_context})
+        language_code = translation.get_language()
+        feed = feedparser.parse(RSS_FEED_URLS[language_code])
+        # select five most recent feeds
+        feed["entries"] = feed["entries"][:5]
+        # decode html entities like dash and split after line break
+        for entry in feed["entries"]:
+            entry["summary"] = html.unescape(entry["summary"]).split("\n")[0]
+        domain = urlparse(RSS_FEED_URLS["home-page"]).netloc
+        # breakpoint()
```

Comment:
```suggestion
```
I think as long as we don't use the python debugger actively in our development tooling, we should remove such comments.